src/components/Indicator/Indicator.test.tsx

```diff
@@ -1,315 +1,154 @@
 import { render } from '@testing-library/react';
 import { checkDanger, Indicator, IndicatorProps } from './Indicator';
 
-describe('Indicator', () => {
-
-  // const checkDanger = (model: IndicatorProps) => {
-  //   if (model.alarm && model.alarm.values) {
-  //     return model.value > model.alarm.values[0];
-  //   }
-  // }
-
-  const modelIndicator: IndicatorProps = {
-    value: 30,
-    unit: 'graus',
-    name: 'temperatura',
-    alarm: {
-      condition: '>',
-      values: [29]
-    }
-  };
-
+const modelIndicator: IndicatorProps = {
+  value: 30,
+  unit: 'graus',
+  name: 'temperatura',
+  alarm: {
+    condition: '>',
+    values: [29]
+  }
+};
+
+describe('Indicator defaults tests', () => {
   it('should create the indicator on screen', () => {
     render(<Indicator config={modelIndicator} />);
   })
 
-  it('idicator without alarm >', () => {
+  it('idicator without alarm', () => {
     const withoutAlarm: IndicatorProps = {
       value: 27,
       unit: 'graus',
       name: 'temperatura'
     };
     expect(checkDanger(withoutAlarm)).toBeFalsy();
```

Comment:
my personal choice, I avoid using toBeFalsy and toBeTruthy. The reason: values like null, undefined and {} can be confusing :(